include/nccl-headers/nvidia/tuner.h

```diff
@@ -12,7 +13,17 @@
 #include "nccl-headers/error.h"
 
 #define NCCL_NUM_FUNCTIONS 5 // Send/Recv not included for now
-typedef enum { ncclFuncBroadcast, ncclFuncReduce, ncclFuncAllGather, ncclFuncReduceScatter, ncclFuncAllReduce, ncclFuncSendRecv, ncclFuncSend, ncclFuncRecv, ncclNumFuncs} ncclFunc_t;
+typedef enum {
+  ncclFuncBroadcast = 0,
+  ncclFuncReduce = 1,
+  ncclFuncAllGather = 2,
+  ncclFuncReduceScatter = 3,
+  ncclFuncAllReduce = 4,
+  ncclFuncSendRecv = 5,
+  ncclFuncSend = 6,
+  ncclFuncRecv = 7,
+  ncclNumFuncs = 8
+} ncclFunc_t;
```

Comment:
If this is cosmetic, leave it unchanged  as NCCL had it.

Reply:
no, this is making it exactly like NCCL has it
https://github.com/NVIDIA/nccl/blob/master/ext-tuner/example/nccl/tuner.h#L17-L28